docs/guide/content_rendering.md

```diff
@@ -14,26 +14,19 @@ It can also be called straight by its direct URI: 
 
 A Content item can also have different **view types** (full page, abstract in a list, block in a landing page, etc.). By default the view type is **full** (for full page), but it can be anything (*line*, *block, etc*.).
 
-## Configuration
+## Configuring views: the ViewProvider
 
-### View provider configuration
-
-The **configured ViewProvider** allows you to configure template selection when using the `ViewController`, either directly from a URL or via a sub-request.
-
-eZ Publish 4.x terminology
-
-In eZ Publish 4.x, it was known as **template override system *by configuration*** (`override.ini`).
-However this only reflects old overrides for `node/view/*.tpl` and `content/view/*.tpl`.
+The **ViewProvider** allows you to configure template selection when using the `ViewController`, either directly from a URL or via a sub-request.
 
 #### Principle
 
-The **configured ViewProvider** takes its configuration from your siteaccess in the `content_view` section. This configuration is a hash built in the following way:
+The ViewProvider takes its configuration from your SiteAccess in the `content_view` section. This configuration is a hash built in the following way:
```

Comment:
If there is none configured, what's happening then?

Reply:
If there's no config under `content_view`, there will simply be no views defined, and configuring views is described below in https://github.com/ezsystems/developer-documentation/pull/74/files#diff-f6648de0e09ac6bb97159196e4fcc3d6R7
Do you think we should also mention it here?